### ISSUE TITLE
Clean-css error while building an app: Broken @imports with relative paths

### ISSUE THREAD

**Issue by ArkhipovK** [driftyco/ionic-app-scripts#712]
Hi guys, i have a problem with building an application via ">ionic build android --prod" command. The cause of the problem is clean-css module, which throws an error on relative paths in @import statements.

![snapshot_170207_142322](https://cloud.githubusercontent.com/assets/9945452/22685195/37372730-ed41-11e6-9dd5-e40422f1f298.png)

The @imports were used:

```
    @import '../assets/fonts/Roboto/css/Roboto.css';
    @import '../assets/fonts/RobotoCondensed/css/RobotoCondensed.css';
```

The hierarchy of the www folder:

    \---www
         \---assets
         |   \---fonts
         |       \---Roboto
         |           \---css
         |                  Roboto.css
         |           \---fonts
         |                  ......
         |                  Roboto-Fonts.eot
         |                  ......          
         \---build
                 main.js
                 main.css
                 ....

What i tried to do: 
1. Make absolute paths in imports:  @import '/www/assets/fonts/Roboto/css/Roboto.css'; This builds without error at least.
2. Make custom config for this module as here https://ionicframework.com/docs/v2/resources/app-scripts/
    Then, in custom config i tried to use "rebaseTo" option:
 ```
  module.exports = {
      /**
       * sourceFileName: the file name of the src css file
       */
      sourceFileName: process.env.IONIC_OUTPUT_CSS_FILE_NAME,

      /**
       * destFileName: the file name for the generated minified file
       */
      destFileName: process.env.IONIC_OUTPUT_CSS_FILE_NAME,

      rebaseTo: "/www/build"

    };
```
But it had no effect at all.  I tried various combinations, e.g:

main.css:

    @import 'assets/fonts/Roboto/css/Roboto.css';
    @import 'assets/fonts/RobotoCondensed/css/RobotoCondensed.css';

config:

    rebaseTo: "/www"

**or**

main.css:

    @import './assets/fonts/Roboto/css/Roboto.css';
    @import './assets/fonts/RobotoCondensed/css/RobotoCondensed.css';

config:

    rebaseTo: "../../../../www"

**Comment by ArkhipovK** [comment]
Ok, while i were working on the problem, i noticed, that i'd been using outdated app-scripts version 0.47. Therefore i'd updated it to current 1.0.0 version. Another mistake is that i was trying to configurate clean-css as 4.x version, but in latest version of app scripts it's version is 3.x. Also i figured out that i was using wrong configuration structure.

Finally, i've gotten some results, after solving all of these mistakes.
With option that allows to ignore the @imports, my biuld was succeeded:

```
module.exports = {
    options:{
      processImport:false
  }
};
```
**but**
	
with 'relativeTo' option, i still get an error

```
module.exports = {
    options:{
      root:"/www",

      relativeTo: "/www/build",

      target: "/www/build/main.css"
  }
};
```
![snapshot_170210_141111](https://cloud.githubusercontent.com/assets/9945452/22821044/34c7cf20-ef9b-11e6-8b89-7da4dda7aadf.png)

What i'm doing wrong?

**Comment by ArkhipovK** [comment]
Ok. Working on the promlem a bit more time, i figured out, that root for clean-css module is not project folder. After i used environment variables for paths, the build had become succeeded.
```
module.exports = {
    options:{
      //processImport:false
      root:process.env.IONIC_ROOT_DIR+"/www",

      relativeTo: process.env.IONIC_ROOT_DIR+"/www/build",

      target: process.env.IONIC_ROOT_DIR+"/www/build/main.css"
  }
};
```
**Is it able to include such config in master project?**

**Comment by tHesTx** [comment]
What worked for me:
1. @import '/www/assets/animations/animate.css';
2. include directly in **index.html**:  <link href="assets/animations/animate.css" rel="stylesheet">

It's a very big difference in load and run speed(second is better) between: **ionic build android** and **ionic build android --prod**

**Comment by ArkhipovK** [comment]
This will have been working until there is no relative urls inside animate.css
(sorry for ridiculous tense)

**Comment by tHesTx** [comment]
Yep, there's no other import in animate.css - but I've had the same error when building "--prod", for normal build no such error(not sure why)

**Comment by ArkhipovK** [comment]
> for normal build no such error(not sure why)
Obviously because clean-css was included only in production build.

**Comment by huh66** [comment]
Similar problem: Error: Broken @import declaration of "./../assets/css/glyphicons.css"

Adding 

```
module.exports = {
    options:{
      processImport:false
  }
};
```
to ./node_modules/@ionic/app-scripts/config/cleancss.config.js works for me. 
Thanks to ArkhipovK

**Comment by moda20** [comment]
i read the earlier comments and I have a question: 

first : where do i find the cleancss.config.js
